science/nektarpp/Portfile

```diff
@@ -32,30 +32,19 @@ master_sites            ${master_sites}:main \
                         ${homepage}thirdparty/:thirdparty
```

Comment:
it looks like there is some additional clean-up possible now that there is only one distfile left and likely the standard from the `gitlab` PG would suffice?

Reply:
I've left this in place for now, since if one of the `mpi` variants is enabled then we do need a third-party source that is not currently packaged (gsmpi). I have started to look at this, but will be a bit of work to complete it.